osidb/tests/factories.py

```diff
@@ -49,6 +49,15 @@ class Meta:
         yes_declaration=factory.Faker("future_datetime", tzinfo=UTC),
         no_declaration=factory.Faker("past_datetime", tzinfo=UTC),
     )
+    summary = factory.Maybe(
+        "embargoed",
+        yes_declaration=factory.sequence(
+            lambda n: f"EMBARGOED - CVE-2022-000{n} kernel-rt: kernel: race condition in example function [rhel-1.0.0.z]"
+        ),
+        no_declaration=factory.sequence(
```

Comment:
Same case with the `factory.sequence` -> `factory.LazyAttribute`

Reply:
Actually after reading @osoukup comment the `title` which in fact is a `summary` from SFM2 already uses the `factory.LazyAttribute` so you can just tweak it accordingly.

Reply:
makes sense, changed to reflect the suggestion.